modules/@apostrophecms/asset/index.js

```diff
@@ -123,7 +200,9 @@ module.exports = {
             });
           }
 
-          async function moduleOverrides() {
+          async function moduleOverrides(modulesDir, source) {
```

Comment:
Each build needs its own working folder where the assets for that module name get copied in "subclass last" order, so that webpack can resolve things simply against that folder. This isn't new but it's per webpack build now, not just the apos build.